applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_isotropic_damage_3d.cpp

```diff
@@ -20,6 +20,22 @@
 #include "custom_constitutive/generic_small_strain_isotropic_damage_3d.h"
```

Comment:
I think it is not necessary because is already included in generic_small_strain_isotropic_damage_3d.h (duplication of includes increases a lot compilation time)

Reply:
I've added them because in windows it was not compiling... we'll check anyways

Reply:
Ok